.travis.yml

```diff
@@ -29,22 +29,22 @@ env:
     # Test with both clang and gcc
     # Only gcc can test openMP, and travis does not support gcc when gcc uses too much memory
     # So some tests have been commented out, as it is useless to compile them with clang
-    - PYTHON_VERSION=2 PEP8=false TESTCASE=pythran/tests/test_advanced.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-g"
```

Comment:
I though that we had to force the python version, but your usage seems to lead toward : if it's both python2 and Python3 compatible, then no PYTHON_VERSION=XX ?